job_server/src/test/mailingList.spec.ts

```diff
@@ -0,0 +1,65 @@
+import app from "../server";
```

Comment:
could tests be in a file at the same level as src instead of inside src?

Reply:
Had a go at fixing this for a good while but I'm getting a weird typeorm error that I can't seem to figure out. I think it's something to do with creating the connection in the wrong subdirectory or something.